pkg/util/mount/mount.go

```diff
@@ -76,3 +80,40 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	}
 	return refs, nil
 }
+
+// GetDeviceRefCount: given a device path, find its reference count from /proc/mounts
+// returns the reference count to the device and error code
+func GetDeviceRefCount(mounter Interface, deviceName string) (int, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return -1, err
+	}
+
+	// Find the number of references to the device.
+	refCount := 0
+	for i := range mps {
+		if strings.HasPrefix(mps[i].Device, deviceName) {
+			refCount++
+		}
+	}
+	return refCount, nil
+}
+
+// GetDeviceFromMnt: given a mnt point, find the device from /proc/mounts
+// returns the device name, reference count, and error code
+func GetDeviceFromMnt(mounter Interface, mnt string) (string, int, error) {
```

Comment:
This name is not good.  First, it is not descriptive.  Second, no need to abbreviate Mount to Mnt.